phi/tools/slack.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+from phi.tools.toolkit import Toolkit
+from phi.utils.log import logger
+
+
+class SlackTools(Toolkit):
+    def __init__(self, token: str):
+        super().__init__(name="slack")
+        self.client = WebClient(token=token)
+        self.register(self.send_message)
+        self.register(self.list_channels)
+        self.register(self.get_channel_history)
```

Comment:
wrap in variables eg:

if send_message:
    ...
    
see: https://github.com/phidatahq/phidata/blob/main/phi/tools/duckdb.py#L36-L45